ga-ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventGAspec-CUI-nonprod.json

```diff
@@ -291,5 +291,17 @@
         "CRUD": "CRU"
       }
     ]
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseEventID": "CREATE_LIP_APPLICATION",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "citizen-profile"
+        ],
+        "CRUD": "CRU"
+      }
+    ]
```

Comment:
do we still need this event?

Reply:
Changed from CREATE_LIP_APPLICATION to GENERAL_APPLICATION_CREATION, which will need citizen auth.